build-openresty.sh

```diff
@@ -29,14 +29,28 @@ mkdir -p /tmp/build/usr/local/kong
 mkdir -p /tmp/build/
 mkdir -p /work
 
-if [ "$DEBUG" == 1 ]
+if [ -z "$KONG_NGINX_MODULE" ]
 then
-  KONG_NGX_BUILD_ARGS="--debug"
+  KONG_NGINX_MODULE="master"
 fi
 
-if [ -z "$KONG_NGINX_MODULE" ]
+NGX_BUILD_FLAGS=(
+  "--openresty $RESTY_VERSION"
+  "--openssl $RESTY_OPENSSL_VERSION"
+  "--luarocks $RESTY_LUAROCKS_VERSION"
+  "--kong-nginx-module $KONG_NGINX_MODULE"
+  "--pcre $RESTY_PCRE_VERSION"
+  "--work /work"
+)
+
+if [ ! -z "$OPENTRACING_VERSION" ]
```

Comment:
conditional simplification and adding missing nginx module for opentracing; no need to add either build argument if only one is present (this requirement could have an output message too in the initial checks if you prefer)

```suggestion
if [ -n "$OPENTRACING_VERSION" ] && [ -n "$OPENTRACING_NGINX_MODULE" ]
```